ChatterboxBackend/Chatterbox/Controllers/UserController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Chatterbox.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Chatterbox.Controllers
+{
+    [Route("api/[controller]")]
+    public class UserController:Controller
+    {
+        private readonly ChatterContext _context;
+
+        public UserController(ChatterContext context)
+        {
+            _context = context;
+        }
+
+        [HttpPost("login")]
+        public IActionResult LogMeIn([FromBody] LoginDetails details)
+        {
+            Console.WriteLine("NEW LOG",details.Email);
+            var emailCheck = _context.Users.FirstOrDefault(t => t.Email == details.Email);
+
+
+            if (emailCheck == null)
+            {
+                return new ObjectResult(new { status = "failed",message="No such user", user = "" });
+            }
+
+            var items = _context.Users.FirstOrDefault(t => t.Email == details.Email && t.Password == details.Password);
+
+
+            if (items == null)

```

Comment:
Good. This is the correct usage. `FirstOrDefault` will request the data from the database.